### PR TITLE
add parameters 31, 32

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/parameter-types.md [hunk 1]
```diff
@@ -580,3 +580,91 @@ It is a `1`-byte unsigned int.
...
+
+#### ***channels bit set*
+
+[Channels to set configuration](../types.md#channels-bit-set).
+
+#### **pull up time**
+
+Channel pull up time in microseconds.
+<br>
+Minimal value - 17 μs, maximum - 255 μs, 18 μs by default.
```
Comment (DarkPark) [inline docs/parameter-types.md:611]:
```suggestion
Minimal value - `17` `μs`, maximum - `255` `μs`, `18` `μs` by default.
```
?

File: docs/parameter-types.md [hunk 2]
```diff
@@ -580,3 +580,91 @@ It is a `1`-byte unsigned int.
...
+### Format
+
+| Size | Type   | Field                                 |
+| ---- | ------ | ------------------------------------- |
+| 1    | `byte` | parameter type = `31`                 |
+| 1    | `byte` | [channels bit set](#channels-bit-set) |
+| 1    | `byte` | [pull up time](#pull-up-time)         |
+| 1    | `byte` | [scan time](#scan-time)               |
+
+#### ***channels bit set*
```
Comment (DarkPark) [inline docs/parameter-types.md:603]:
```suggestion
#### **channels bit set**
```

File: docs/parameter-types.md [hunk 3]
```diff
@@ -580,3 +580,91 @@ It is a `1`-byte unsigned int.
...
+hardware type - `0x06` hardware version - `0x0a`
+
+[Hardware types](../../basics.md#hardware-types)
+
+### Format
+
+| Size | Type   | Field                                 |
+| ---- | ------ | ------------------------------------- |
+| 1    | `byte` | parameter type = `32`                 |
+| 1    | `byte` | [channels bit set](#channels-bit-set) |
```
Comment (DarkPark) [inline docs/parameter-types.md:653]: wrong link

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: docs/parameter-types.md (hunk 1 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

File: docs/parameter-types.md [hunk 4]
```diff
@@ -580,3 +580,91 @@ It is a `1`-byte unsigned int.
...
+[Hardware types](../../basics.md#hardware-types)
+
+### Format
+
+| Size | Type   | Field                                            |
+| ---- | ------ | ------------------------------------------------ |
+| 1    | `byte` | parameter type = `32`                            |
+| 1    | `byte` | [channels bit set](../types.md#channels-bit-set) |
+
+#### ***channels bit set*
```
Comment (DarkPark) [inline docs/parameter-types.md:655]:
```suggestion
#### **channels bit set**
```

File: docs/parameter-types.md (hunk 3 above)
Comment (DarkPark) [reply docs/parameter-types.md]: no,
it should be something like this

`[channels bit set](#channels-bit-set-1)`

File: docs/parameter-types.md (hunk 1 above)
Comment (DarkPark) [reply docs/parameter-types.md]: there are some more

Comment (DeadByElpy) [reply docs/parameter-types.md]: fixed

**Pull request merged**